### PR TITLE
Added CronExpression::isValidExpression()

### PR DESCRIPTION
**Pull request by aensley**

### CODE REVIEW

**Comment by aensley** [comment]
Oops... I forgot to add a test. Going to do that now.

**Comment by aensley** [comment]
This is my fix for #77.

File: src/Cron/CronExpression.php [hunk 1]
```diff
@@ -73,6 +73,25 @@ public static function factory($expression, FieldFactory $fieldFactory = null)
...
 
     /**
+     * Validate a CronExpression.
+     *
+     * @param string $expression The CRON expression to validate.
+     *
+     * @return bool True if a valid CRON expression was passed. False if not.
+     * @see Cron\CronExpression::factory
+     */
+    public static function isValidExpression($expression = '')
```
Comment (mtdowling) [inline src/Cron/CronExpression.php]: Why allow an empty string as a default argument?

Comment (aensley) [inline src/Cron/CronExpression.php]: Oh, sorry, force of habit to indicate the variable type of the argument. I can remove it if you want.

Comment (mtdowling) [inline src/Cron/CronExpression.php]: Gotcha. Yes, that would be great.

Comment (aensley) [inline src/Cron/CronExpression.php]: I've done that and added it to PR #87.

**Comment by mtdowling** [comment]
thanks!

**Pull request merged**